src/operations/mod.rs

```diff
@@ -28,6 +33,10 @@ impl Operation {
             Operation::Thumbnail(geom) => resize::thumbnail(image, geom),
             Operation::Scale(geom) => resize::scale(image, geom),
             Operation::Sample(geom) => resize::sample(image, geom),
+            Operation::Composite(other_image, _mask) => {
+                let other_image = wm_try!(decode(&other_image.location, other_image.format));
```

Comment:
There's probably a debate to be had on where to decode the second and the mask image. Feels somewhat _wrong_ to decode here!?